doc/api/console.md

```diff
@@ -9,10 +9,14 @@ The module exports two specific components:
 
 * A `Console` class with methods such as `console.log()`, `console.error()` and
   `console.warn()` that can be used to write to any Node.js stream.
-* A global `console` instance configured to write to `stdout` and `stderr`.
-  Because this object is global, it can be used without calling
+* A global `console` instance configured to write to [`process.stdout`][] and
+  [`process.stderr`][].  This object is global, it can be used without calling
   `require('console')`.
 
+***Warning***: The global console object's methods may be synchronous, because
```

Comment:
How about

> Warning: Because the global console object's methods are neither consistently synchronous like the browser APIs they resemble, nor are they consistently asynchronous like all other Node.js streams. See the note ... etc